cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
is this file needed since there is a `cypress.json` in the `cypress` directory?

Reply:
This file is needed to execute the framework... In the cypress directory there is another one to be modified in future only if we add a continuous delivery app like CircleCI, Jenkins...